scripts/items/daybreak.lua

```diff
@@ -12,12 +12,12 @@ end
 itemObject.onItemEquip = function(target, item)
     local mainWeapon = target:getEquippedItem(xi.slot.MAIN)
     if mainWeapon and mainWeapon:getID() == xi.item.DAYBREAK then
-        target:addSpell(xi.magic.spell.DISPELGA, true)
+        target:addSpell(xi.magic.spell.DISPELGA, true, false)
```

Comment:
One million percent we can't continue with this pattern of passing naked bools around as flags. This is a _huge_ pain in the ass, especially when IDE tooling for Lua is so poor/slow. There's two choices here:

1. Make these into enums that have true/false values, and then the C++ size doesn't have to change
2. Pass in a table with named members for the settings

Both of these will require changes to the LLS specs. I don't mind which you go with, but these naked bools aren't the one (yes I'm aware the codebase is _chock full_ of these, but we can do better than the past)